### PR TITLE
add support for phoscon hive

### PR DESCRIPTION
**Pull request by Backup7750**
please review, this is my first addon for zigbee2mqtt and my first commit in github. (worked for me as external addon in zigbee2mqtt)

### CODE REVIEW

File: src/devices/dresden_elektronik.ts [hunk 1]
```diff
@@ -32,6 +32,13 @@ const definitions: Definition[] = [
         extend: extend.light_onoff_brightness(),
         ota: ota.zigbeeOTA,
     },
+    {
+        zigbeeModel: ['Hive'], // The model ID from: Device with modelID 'lumi.sens' is not supported.
+        model: 'Hive', // Vendor model number, look on the device for a model number
```
Comment (Koenkk) [inline src/devices/dresden_elektronik.ts:37]: Is there any model number written on the device or product site? If yes use that here.

Comment (Backup7750) [reply src/devices/dresden_elektronik.ts:37]: no unluciily not. 
via Zigbee2MQTT i can read the following Data
'genBasic': {"modelId":"Hive","manufacturerName":"Phoscon"}
on the website is also no product code
https://phoscon.de/en/hive

File: src/devices/dresden_elektronik.ts [hunk 2]
```diff
@@ -32,6 +32,13 @@ const definitions: Definition[] = [
         extend: extend.light_onoff_brightness(),
         ota: ota.zigbeeOTA,
     },
+    {
+        zigbeeModel: ['Hive'],
+        model: 'Hive',
+        vendor: 'Phoscon',
+        description: 'Battery powered smart LED light',
+        extend: extend.light_onoff_brightness_colortemp_color(),
```
Comment (Koenkk) [inline src/devices/dresden_elektronik.ts:40]: Please add the color temperature range as described on https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html#_3-adding-converter-s-for-your-device

**Comment by Backup7750** [comment]
How can I add also the Power to the Template with the extend method?

i can read out the following function via Zigbee2mqtt
Read result of 'genPowerCfg': {"batteryPercentageRemaining":80}

**Comment by Koenkk** [comment]
Updated the code to add this, please check if it works (https://github.com/Koenkk/zigbee-herdsman-converters/pull/5902/files)

**Comment by Backup7750** [comment]
got the same error like in ci. zigbee2mqtt will not start.

/app/node_modules/zigbee-herdsman-converters/index.js:58
            assert.fail(`'${definition.model}' has configure in extend and device, this is not allowed`);
                   ^
AssertionError [ERR_ASSERTION]: 'Hive' has configure in extend and device, this is not allowed
    at Object.addDefinition [as addDeviceDefinition] (/app/node_modules/zigbee-herdsman-converters/index.js:58:20)
    at new ExternalConverters (/app/lib/extension/externalConverters.ts:15:17)
    at new Controller (/app/lib/controller.ts:84:58)
    at start (/app/index.js:106:18)

but can check this only as external converter.

**Comment by Koenkk** [comment]
Fixed that, updated the code

**Comment by Backup7750** [comment]
battery and lamp works fine now. Thanks a lot!
but device does not appear in update list.

**Comment by Koenkk** [comment]
What do you mean with "update list"?

**Comment by Backup7750** [comment]
in the panel "OTA" All devices are shown up which can be updated then you can search for firmware there.
what can I do to check if an update for this device is available if not in that "list"?

**Comment by Koenkk** [comment]
Are you sure you added the ` ota: ota.zigbeeOTA,` line? If this is present it will show up under OTA updates.

**Comment by Backup7750** [comment]
okay this line i missed. now it works... my fault :D

**Comment by Koenkk** [comment]
Thanks!

**Comment by Backup7750** [comment]
welcome
and a big thank you to you and this great project and the great support for that!

**Pull request merged**